lib/bundler/uri_credentials_filter.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module Bundler
+  module URICredentialsFilter
+    def anonymized_uri(uri_to_anonymize)
+      return uri_to_anonymize if uri_to_anonymize.nil?
+      uri = uri_to_anonymize.dup
+      uri = URI(uri.to_s) unless uri.is_a?(URI)
+      uri.user = uri.password = nil if uri.userinfo
```

Comment:
Asking because I feel that user still fits the category of a credential when it comes to authentication.